app/views/searches/show.html.haml

```diff
@@ -26,17 +26,17 @@
               %p= t(".data_sources_preface", count: number_with_delimiter(@legal_entity_count))
               %ul.list-unstyled
                 - if @data_sources['uk-psc-register'].present?
-                  %li= link_to('UK PSC Register', data_source_path(@data_sources['uk-psc-register']))
+                  %li= link_to('UK PSC Register', data_source_path(@data_sources['uk-psc-register'].slug))
```

Comment:
Defining `def to_param; slug; end` as noted in https://github.com/openownership/register/issues/168 would have meant these changes weren't needed, but it's fine.